### PR TITLE
Add PR number badge for internal builds

### PR DESCRIPTION
**Pull request by staszekscp**
### Details

This PR customises the environmental badge for internal builds. The internal builds are going to run on staging environment, but there will be a badge with information from which PR the app was built.

The PR doesn't include screenshots, since a real internal builds would have to be checked to test it properly.

@AndrewGable Could you run the `testBuild` workflow against this PR to see if everything is fine?

### Fixed Issues

$ https://github.com/Expensify/App/issues/13993

### Tests

- [x] Verify that no errors appear in the JS console
- [x] Check if the correct badge is visible near "Chats", when building the app directly from a PR

### QA Steps

- [x] Verify that no errors appear in the JS console
- [x] On Staging "Stg" badge should be visible, on production no badge expected

### PR Author Checklist

- [x]  I linked the correct issue in the `### Fixed Issues` section above
- [x]  I wrote clear testing steps that cover the changes made in this PR
    - [x]  I added steps for local testing in the `Tests` section
    - [x]  I added steps for the expected offline behavior in the `Offline steps` section
    - [x]  I added steps for Staging and/or Production testing in the `QA steps` section
    - [x]  I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x]  I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x]  I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x]  I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x]  I ran the tests on **all platforms** & verified they passed on:
    - [x]  Android / native
    - [x]  Android / Chrome
    - [x]  iOS / native
    - [x]  iOS / Safari
    - [x]  MacOS / Chrome / Safari
    - [x]  MacOS / Desktop
- [x]  I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x]  I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x]  I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x]  I verified that comments were added to code that is not self explanatory
    - [x]  I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x]  I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x]  If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x]  I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x]  I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x]  I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x]  I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x]  If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x]  I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x]  I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x]  I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x]  I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x]  I verified that if a function's arguments changed that all usages have also been updated correctly
- [x]  If a new component is created I verified that:
    - [x]  A similar component doesn't exist in the codebase
    - [x]  All props are defined accurately and each prop has a `/** comment above it */`
    - [x]  The file is named correctly
    - [x]  The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x]  The only data being stored in the state is data necessary for rendering and nothing else
    - [x]  For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x]  Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x]  All JSX used for rendering exists in the render method
    - [x]  The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x]  If any new file was added I verified that:
    - [x]  The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x]  If a new CSS style is added I verified that:
    - [x]  A similar style doesn't already exist
    - [x]  The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x]  If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x]  If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x]  If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x]  If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x]  If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x]  I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

image

</details>

<details>
<summary>Mobile Web - Chrome</summary>

IMG_2056

</details>

<details>
<summary>Mobile Web - Safari</summary>

IMG_2057

</details>

<details>
<summary>Desktop</summary>

image

</details>

<details>
<summary>iOS</summary>

IMG_2055

</details>

<details>
<summary>Android</summary>

rn_image_picker_lib_temp_f95cde4c-9398-4433-b764-3b159164fadd

</details>

### CODE REVIEW

File: .github/workflows/testBuild.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || needs.getBranchRef.outputs.REF }}
 
+      - name: Add PULL_REQUEST_NUMBER to .env.staging file
```
Comment (AndrewGable) [inline .github/workflows/testBuild.yml:74]: I can appreciate the simplicity of this but I wonder if creating a `.env.adhoc` is more clear?

Comment (staszekscp) [reply .github/workflows/testBuild.yml:74]: I took that into consideration, but quoting Rory from [this discussion](https://github.com/Expensify/App/issues/13993#issuecomment-1378062235) :)

> we could create a separate environment file for ad-hoc builds. The downside is that we'd be introducing a new environment file that we have to maintain, and that environment file would be different from the staging environment. In general I think we want these ad-hoc builds to be as similar to staging builds as possible.

Comment (AndrewGable) [reply .github/workflows/testBuild.yml:74]: @roryabraham - Curious for your thoughts here now that we have a PR up. My sense is adhoc is not a ton to maintain and it's more explicit as we now have two or three different terms to describe "adhoc" "pull request" "internal testing", etc.

**Comment by AndrewGable** [comment]
I can probably checkout the branch, then push my own version?

**Comment by staszekscp** [comment]
Yep, I think that would be the easiest solution!

**Comment by staszekscp** [comment]
@AndrewGable There was a tiny bug I haven't spotted before. The `.env.staging` file was not edited in the previous build, because double quotes  in `echo` command ended in the wrong place... Could you pull  the newest changes and run the workflow again? Sorry for that, and thanks for the help!

**Comment by AndrewGable** [comment]
Yes, will do. Updated `internal-build-pr-badge` branch and will re-run now.

**Comment by AndrewGable** [comment]
Build: https://github.com/Expensify/App/actions/runs/4598086291

**Comment by staszekscp** [comment]
Hi @AndrewGable! Thanks for running the workflow, it seems that everything works as expected :)

**Comment by AndrewGable** [comment]
From the QR code I still see the `stg` badge when logging in: 

IMG_E01FE9C6313C-1

**Comment by AndrewGable** [comment]
Also the "About" page from settings still shows the non adhoc version

IMG_AAC2EB098BB5-1

**Comment by staszekscp** [comment]
@AndrewGable, sorry about the About page, I didn't know it has to be changed there too. It won't be a problem, I'll fix it tomorrow. If it comes to the badge on the screen with logging in I can change it. I just want to inform that for now it is an `svg`, so I'll have to modify this element a bit to use the `EnvironmentBadge` component instead. Although if you'd like to modify changes the SplashScreen too, the implementation may be more complex :)

**Comment by AndrewGable** [comment]
I think no splash screen is fine for now, I can get you a replacement SVG for AdHoc that looks similar to the Staging one.

**Comment by staszekscp** [comment]
@AndrewGable That would be great!

**Comment by AndrewGable** [comment]
@Expensify/design or @ADeWitt - Would it be possible to get https://github.com/Expensify/App/blob/main/assets/images/new-expensify-dev.svg in another variant? Ideally it would say "AdHoc", if that does not fit, maybe we could do "PR"? The "pill" color could be a different color if we have another brand color to use, otherwise we can use the "Dev" color.

**Comment by shawnborton** [comment]
Do we need that for all of the icon possibilities? Like the desktop icon as well?

**Comment by AndrewGable** [comment]
Going to let @staszekscp confirm, but I think yes.

**Comment by staszekscp** [comment]
@AndrewGable @shawnborton Yes, I think it we could swap the `stg` badge with `adhoc` wherever it is necessary :)

**Comment by AndrewGable** [comment]
It's failing to build, looking into it

**Comment by AndrewGable** [comment]
New build: https://github.com/Expensify/App/actions/runs/4672621863

**Comment by staszekscp** [comment]
Hi @AndrewGable! I see the previous build failed, I suppose git history was not correctly aligned between your branch and my PR... Could you have a look at it?

**Comment by AndrewGable** [comment]
Build here (I think it's working): https://github.com/Expensify/App/actions/runs/4690785038

**Comment by Julesssss** [comment]
I'm going to fully test this tomorrow, but I'm curious why we're not creating the Android app icon with the built-in Android Studio Image Asset tool? This will ensure the icon looks correct on all Android versions and ROMS.

**Comment by AndrewGable** [comment]
@staszekscp - Can you check if this desktop build error is related to these changes?

```
• electron-builder  version=23.5.0 os=21.6.0
  ⨯ Invalid ELECTRON_ENV!  failedTask=build stackTrace=Error: Invalid ELECTRON_ENV!
    at Object.<anonymous> (/Users/runner/work/App/App/config/electronBuilder.config.js:31:11)
    at Module._compile (node:internal/modules/cjs/loader:[110](https://github.com/Expensify/App/actions/runs/4690785038/jobs/8316839984#step:7:111)5:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at readConfig (/Users/runner/work/App/App/node_modules/read-config-file/src/main.ts:19:14)
    at Object.getConfig (/Users/runner/work/App/App/node_modules/app-builder-lib/src/util/config.ts:44:34)
    at Packager.build (/Users/runner/work/App/App/node_modules/app-builder-lib/src/packager.ts:314:27)
    at Object.executeFinally (/Users/runner/work/App/App/node_modules/builder-util/src/promise.ts:12:14)
```

**Comment by AndrewGable** [comment]
Trying again here: https://github.com/Expensify/App/actions/runs/4692202389

**Comment by staszekscp** [comment]
Thanks for running the builds! I think I've just fixed the last issue with badge color and everything should work fine :) Additionally I'm going to add different icons and splash screens for the AdHoc build in [this PR](https://github.com/Expensify/App/pull/17265), but I think here we're ready to go :) If you'd like to see the correct badge color you can run the build one more time :)

**Comment by Julesssss** [comment]
>  I'm curious why we're not creating the Android app icon with the built-in Android Studio Image Asset tool? 

Oh I see, the icon is only shown within the app currently... Can we make is show in the OS home page too? We can use the existing tools to create these and it will help simplify switching between apps varients locally.

**Review by Julesssss — APPROVED** [review]
Nice work. This is looking good, but could we update the app launcher icons too?

**Comment by staszekscp** [comment]
Thanks @Julesssss! I'm going to apply separate icons while working on flavors - I don't want to do it in this PR, because it would have to be solved differently. Therefore I would like to avoid adding this feature in this PR :)

**Comment by Julesssss** [comment]
Cool thanks, sounds fine to me.

**Review by Julesssss — COMMENTED** [review]
I'm seeing a build error with Desktop [here](https://github.com/Expensify/App/actions/runs/4699158808/jobs/8332320138#step:6:16).

**Review by Julesssss — CHANGES_REQUESTED** [review]
I'm seeing a build error with Desktop [here](https://github.com/Expensify/App/actions/runs/4699158808/jobs/8332320138#step:6:16).

**Comment by staszekscp** [comment]
It seems that wrong file was taken to execute workflow - have you pulled the newest changes from this branch? The problem was fixed in this [commit](https://github.com/Expensify/App/pull/16735/commits/0bbe8a110338d8e51f91a2eac0451173f769455b) Maybe accidently you've taken wrong branch to execute the workflow?

**Comment by staszekscp** [comment]
I see that in Andrew's run 17 hours ago everything was built correctly, and the only thing I changed since then were badge colors: https://github.com/Expensify/App/pull/16735/commits/30fe055ff5018cb31180bfcb2b3304b5f6aa909c

**Comment by Julesssss** [comment]
hey @staszekscp maybe I'm missing something, but it looks like my PR include the last commit `30fe055ff5018cb31180bfcb2b3304b5f6aa909c`? (fix badge colour)

Screenshot 2023-04-14 at 12 49 57

**Comment by staszekscp** [comment]
Hmm... Have you chosen `Branch: jules-test16735` instead of `main` here?

image

**Comment by Julesssss** [comment]
> Hmm... Have you chosen Branch: jules-test16735 instead of main here?

Yep, that's it. Trying again [here](https://github.com/Expensify/App/actions/runs/4700442827)

**Comment by staszekscp** [comment]
@Julesssss I think it failed already 😄 The number in the input should be 16735, I think you've typed 17453 😄

**Comment by Julesssss** [comment]
[3rd time lucky](https://github.com/Expensify/App/actions/runs/4700526389) 🤞

**Comment by staszekscp** [comment]
Good news - seems to work fine :)

**Review by AndrewGable — DISMISSED** [review]
All good from me, feel free to merge @Julesssss

**Comment by staszekscp** [comment]
Hey @Julesssss! I had to resolve a small conflict, and now the PR is ready to merge :)

**Review by Julesssss — DISMISSED** [review]
👍

**Review by AndrewGable — DISMISSED** [review]

**Comment by Julesssss** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Comment by staszekscp** [comment]
The desktop theoretically should be fixed, but we would have to run a build to make sure if everything was fine :) The `__DEV__` env is a RN variable, but it has to be set manually by webpack for the desktop app. I haven't noticed that there is a regex that parses the .env file to decide what should be the value of `__DEV__`.

**Comment by Julesssss** [comment]
I kicked off a new ci build[ here](https://github.com/Expensify/App/actions/runs/4732816715).

**Comment by staszekscp** [comment]
Hey thanks for the run! Actually it already failed, because the PR number in the input was 17573 instead of 16735 :) An alternative solution would be to add `Ready to build` to your PR :)

**Comment by Julesssss** [comment]
> Actually it already failed, because the PR number in the input was 17573 instead of 16735 :)

Oh I made the same mistake last time 😁

**Comment by Julesssss** [comment]
Maybe we can simplify the CI input labels? Even with context I've messed up twice now. 

New run: https://github.com/Expensify/App/actions/runs/4734387668

**Comment by staszekscp** [comment]
Hi @Julesssss, thanks for the run! I've just merged the newest `main`, and the desktop app works, so hopefully it can be finally merged :) Fingers crossed!

**Comment by Julesssss** [comment]
I'm just a bit confused why the Android AdHoc build doesn't contain the purple pill like the other platforms. I can only see the PR info on the settings page 😕 

I used this link, from [here](https://github.com/Expensify/App/pull/16735#issuecomment-1513505353) is that correct? 

Screenshot_20230419-123415

**Comment by Julesssss** [comment]
Thanks, triggered another build [here](https://github.com/Expensify/App/actions/runs/4743324720)

**Comment by staszekscp** [comment]
@Julesssss all of them seem to be working fine now 😄

**Comment by Julesssss** [comment]
Okay, thanks. Confirmed I see the pill now. 

I've updated the checklist will all platforms except iOS mWeb -- which I'm struggling with for some reason. Both in Safari and Chrome: 


IMG_0130 IMG_0131

**Comment by staszekscp** [comment]
That is strange, on my iPhone everything works just fine... I've just double checked it to make sure. Could you try a bit later?

**Comment by staszekscp** [comment]
One screenshot is from Safari, second from Chrome

IMG_2101
IMG_2102

File: src/components/EnvironmentBadge.js [hunk 2]
```diff
@@ -16,12 +19,15 @@ const EnvironmentBadge = (props) => {
         return null;
     }
 
+    const text = Environment.isInternalTestBuild() ? `${pkg.version}-PR.${CONST.PULL_REQUEST_NUMBER}` : ENVIRONMENT_SHORT_FORM[props.environment];
```
[the remark this answers is not in the document]
Comment (staszekscp) [reply src/components/EnvironmentBadge.js]: I've just pushed a commit with this improvement :)

**Review by Julesssss — APPROVED** [review]

**Comment by Julesssss** [comment]
@AndrewGable maybe you wanna review again

**Comment by mvtglobally** [comment]
@staszekscp @AndrewGable We have no easy way to confirm variety of build for this PR. Do you mind QA-ing it internally?

**Comment by AndrewGable** [comment]
Yes, you can mark it off. We tested the steps for this PR before merging.

**Pull request merged**